c2corg_ui/views/document.py

```diff
@@ -221,3 +222,13 @@ def _diff(self):
             return self.template_input
 
         raise HTTPNotFound()
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
```

Comment:
maybe it would make more sense indeed. But where should i put it ? copy pasting it into init.py gives me this error : `Unbound method calls are not supported, please set the class as your view and the method as your attr
`